causallib/time_varying/GFormula.py

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/env python3
+
+import abc
+from typing import Optional, Any, OrderedDict
+import pandas as pd
+from ..estimation.base_estimator import IndividualOutcomeEstimator
```

Comment:
Use consistent way of importing modules. e.g. if you are using absolute imports be consistent about it

Reply:
got it.